### ISSUE TITLE
show widget on input focus even with appended icon

### ISSUE THREAD

**Issue by fxck** [jdewit/bootstrap-timepicker#108]
now when you use the appended icon, widget only shows on when clicking on the icon..

I know I could do something like `element.focus(function() { element.timepicker("showWidget")});` but I think it could be a feature or an option..

**Comment by frahugo** [comment]
I agree... It should be consistent with bootstrap-datepicker.  Clicking on a span with time icon should not show the widget.  It should do it only if it is a button instead of a span.
